index.html

```diff
@@ -5,11 +5,11 @@
 </style>
 <canvas></canvas>
 
-<script src="js/utils.js"></script>
 <script src="js/collisions.js"></script>
 <script src="js/collisionblock.js"></script>
 <script src="js/Sprite.js"></script>
 <script src="js/Player.js"></script>
 <script src="js/evenListeners.js"></script>
+<script src="js/utils.js"></script>

```

Comment:
Imports turn out not to be required here - all the code gets loaded via the script tags into one "namespace"

However, any code that refers to another function or class or whatever has to be *after* the thing that defines those.  So I needed to move utils.js here